joss/paper.md

```diff
@@ -86,7 +86,7 @@ both as part of an autograder utility for finite-state automata created by stude
 
 ![A visualization of `target_words_dfa`. Transitions on characters leading to immediate rejections are omitted.\label{fig:target_words_dfa}](finite_language_dfa.png){ width=100% }
 
-The following example is inspired by the use case described in [@Johnson_2010].
+The following example is inspired by the use case described in @Johnson_2010.
```

Comment:
What exactly does this change do to the paper? Happy to merge but just curious.

Reply:
Ahhh I see, it includes the name. I think it looks better and will merge!

Reply:
right - we use []s when it's a parenthetical reference, and no []s when we actually want to say something about the work directly.